python-frontend/src/main/java/org/sonar/python/semantic/v2/FunctionTypeBuilder.java

```diff
@@ -52,7 +54,7 @@ public class FunctionTypeBuilder implements TypeBuilder<FunctionType> {
   private boolean hasDecorators;
   private boolean isInstanceMethod;
   private PythonType owner;
-  private PythonType returnType = PythonType.UNKNOWN;
+  private TypeWrapper returnType = new SimpleTypeWrapper(PythonType.UNKNOWN);
```

Comment:
```suggestion
  private TypeWrapper returnType = TypeWrapper.UNKNOWN_TYPE_WRAPPER;
```

There is a wrapper for this one now.